test/e2e/specs/settings.spec.ts

```diff
@@ -0,0 +1,217 @@
+
+      await expectRadBinaryNotFoundToBeVisible(workbench)
+    })
+
+    it('recognizes the rad binary when a valid path is specified', async () => {
+      await $`cp -r ${nodeHomePath} ${tempNodeHomePath}`
+
+      await setTextSettingValue(pathToRadBinarySetting, `/tmp`)
+
+      await expectRadBinaryNotFoundToBeVisible(workbench)
+
+      await setTextSettingValue(pathToRadBinarySetting, `${tempNodeHomePath}/bin/rad`)
+
+      await expectStandardSidebarViewsToBeVisible(workbench)
+
+      await $`rm -rf ${tempNodeHomePath}`
+    })
+
+    // This functionality does not seem to work
+    // eslint-disable-next-line max-len
+    it.skip('recognizes if the directory is created *after* the setting is updated', async () => {
+      await setTextSettingValue(pathToRadBinarySetting, tempNodeHomePath)
+
+      await expectRadBinaryNotFoundToBeVisible(workbench)
+
+      await $`cp -r ${nodeHomePath} ${tempNodeHomePath}`
+
+      await expectStandardSidebarViewsToBeVisible(workbench)
+
+      await $`rm -rf ${tempNodeHomePath}`
+    })
+  })
+
+  /**
+   * In Linux CI:
+   * - The extension is having issues resolving the correct node home directory (RAD_HOME).
+   * - Even when node home is set explicitly in the settings, the extension incorrectly reports it
+   *   as non-authenticated.
+   */
+  // eslint-disable-next-line max-len
+  describe('VS Code, when updating the "Path to Radicle to Node Home" setting, @skipLinuxCI', () => {
+    let pathToNodeHomeSetting: Setting
+    let outputView: OutputView
+
+    before(async () => {
+      pathToNodeHomeSetting = await settings.findSetting(
+        'Path To Node Home',
+        'Radicle',
+        'Advanced',
+      )
+      await workbench.executeCommand('Show Everything Logged in the Output Panel')
+      outputView = await workbench.getBottomBar().openOutputView()
+    })
+
+    after(async () => {
+      await clearInput(await getSettingsSearchBox(settings))
+    })
+
+    afterEach(async () => {
+      await outputView.clearText()
+      await clearTextSetting(pathToNodeHomeSetting)
+
+      await expectRadicleIdentityToBeFound(outputView)
+    })
+
+    it('logs an error in the output console if the path is invalid', async () => {
+      await outputView.clearText()
+
+      await setTextSettingValue(pathToNodeHomeSetting, '/tmp')
+
+      await expectRadicleProfileNotToBeFound(outputView)
+    })
+
+    it('recognizes when a valid path is specified', async () => {
+      await $`cp -r ${nodeHomePath} ${tempNodeHomePath}`
+      await outputView.clearText()
+
+      await setTextSettingValue(pathToNodeHomeSetting, tempNodeHomePath)
+
+      await expectRadicleIdentityToBeFound(outputView)
+
+      await $`rm -rf ${tempNodeHomePath}`
+    })
+
+    // This functionality does not seem to work
+    // eslint-disable-next-line max-len
+    it.skip('recognizes if the directory is created *after* the setting is updated', async () => {
```

Comment:
```suggestion
    it.skip('recognizes if the directory is created *after* the setting is set', async () => {
```

Minor copy improvement. At least for my brain, makes the case description a tad easier to parse.

Reply:
Done!